--- a/test/meson.build	2024-11-14 08:46:37.915088878 -0500
+++ b/test/meson.build	2024-12-23 14:40:53.549304381 -0500
@@ -11,21 +11,6 @@
     protocol: 'exitcode'
     )
 
-if (ascli_exe.found())
-    metainfo_test_file = 'org.naev.Naev.metainfo.xml'
-    test('validate_metainfo',
-    ascli_exe,
-    args: [
-        'validate',
-        '--no-net',
-        '--pedantic',
-        '--verbose',
-        metainfo_test_file
-    ],
-    workdir: meson.project_source_root()
-    )
-endif
-
 if (luacheck.found())
     env = environment()
     env.set( 'LUA_PATH', meson.project_source_root() / '?.lua;;' )
