--- a/meson.build 2023-04-29 22:07:47.096217100 -0400
+++ b/meson.build        2023-04-29 23:00:31.575749605 -0400
@@ -216,7 +216,7 @@
    endif

    # Appstream (Used for generating desktop files and verifying metadata)
-   ascli_exe = find_program('appstreamcli', version: '>=0.12.9', required: false)
+   #ascli_exe = find_program('appstreamcli', version: '>=0.12.9', required: false)

    # Luacheck (linter) wrapper
    luacheck = find_program('luacheck', required: false)
@@ -403,18 +403,18 @@
    if host_machine.system() not in ['windows', 'cygwin', 'emscripten', 'android', 'darwin']
       metainfo_file = 'org.naev.Naev.metainfo.xml'

-      if (ascli_exe.found())
-         # Create desktop-entry file from metainfo
-         custom_target('gen-desktop-entry',
-            input : [metainfo_file],
-            output : ['org.naev.Naev.desktop'],
-            command : [ascli_exe, 'make-desktop-file', '@INPUT@', '@OUTPUT@'],
-            install: true,
-            install_dir: join_paths (get_option ('datadir'), 'applications')
-         )
-      else
-         install_data('org.naev.Naev.desktop', install_dir: join_paths (get_option ('datadir'), 'applications'))
-      endif
+      #if (ascli_exe.found())
+      #   # Create desktop-entry file from metainfo
+      #   custom_target('gen-desktop-entry',
+      #      input : [metainfo_file],
+      #      output : ['org.naev.Naev.desktop'],
+      #      command : [ascli_exe, 'make-desktop-file', '@INPUT@', '@OUTPUT@'],
+      #      install: true,
+      #      install_dir: join_paths (get_option ('datadir'), 'applications')
+      #   )
+      #else
+      install_data('org.naev.Naev.desktop', install_dir: join_paths (get_option ('datadir'), 'applications'))
+      #endif

       install_data(metainfo_file, install_dir: get_option('datadir') / 'metainfo')
