--- a/src/naev.rs	2026-01-03 04:44:15.000000000 -0500
+++ b/src/naev.rs	2026-01-08 21:08:06.265822347 -0500
@@ -294,7 +294,7 @@
         }
 
         //Have to set up fonts before rendering anything.
-        let font_prefix = naevc::FONT_PATH_PREFIX as *const u8 as *const i8;
+        let font_prefix = naevc::FONT_PATH_PREFIX as *const u8 as *const c_char;
         let font_default_path = gettext(
             "Cabin-SemiBold.otf,NanumBarunGothicBold.ttf,SourceCodePro-Semibold.ttf,IBMPlexSansJP-Medium.otf",
         );
