--- a/meson.build	2024-12-23 14:37:36.416070913 -0500
+++ b/meson.build	2024-12-23 15:58:18.267969153 -0500
@@ -111,7 +111,7 @@
    yaml = dependency('yaml-0.1', required: true, version: '>=0.2.5', fallback: ['libyaml', 'yaml_dep'], static: get_option('steamruntime'))
 
    # TODO replace nfd with SDL3 native functions when possible
-   nfd = dependency('nativefiledialog-extended', required: true, fallback: ['nativefiledialog-extended'], static: true)
+   nfd = dependency('nfd', required: true, fallback: ['nativefiledialog-extended'], static: true)
 
    libxml2 = dependency('libxml-2.0', required: false)
    if not libxml2.found()
